### PR TITLE
readme: github pull-requests is the preferred way to contribute

### PR DESCRIPTION
**Pull request by avagin**
We will continue accepting patches.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Here are some useful hints to get involved.
 * CRIU does need [extensive testing](https://github.com/checkpoint-restore/criu/issues?q=is%3Aissue+is%3Aopen+label%3Atesting);
 * Documentation is always hard, we have [some information](https://criu.org/Category:Empty_articles) that is to be extracted from people's heads into wiki pages as well as [some texts](https://criu.org/Category:Editor_help_needed) that all need to be converted into useful articles;
 * Feedback is expected on the github issues page and on the [mailing list](https://lists.openvz.org/mailman/listinfo/criu);
-* For historical reasons we do not accept PRs, instead [patches are welcome](http://criu.org/How_to_submit_patches);
+* We accept github pull-requests and this is the preferred way to contribute into CRIU. If you prefer to send patches by email, you are welcome to send them into [the devel list](http://criu.org/How_to_submit_patches);
```
Comment (adrianreber) [inline README.md]: I would replace both 'into' with 'to'.

**Comment by rst0git** [comment]
Perhaps we should also mention about https://gitter.im/save-restore/CRIU somewhere?

**Comment by adrianreber** [comment]
> Perhaps we should also mention about https://gitter.im/save-restore/CRIU somewhere?

Yes, I was not aware of it.

File: README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Here are some useful hints to get involved.
 * CRIU does need [extensive testing](https://github.com/checkpoint-restore/criu/issues?q=is%3Aissue+is%3Aopen+label%3Atesting);
 * Documentation is always hard, we have [some information](https://criu.org/Category:Empty_articles) that is to be extracted from people's heads into wiki pages as well as [some texts](https://criu.org/Category:Editor_help_needed) that all need to be converted into useful articles;
 * Feedback is expected on the github issues page and on the [mailing list](https://lists.openvz.org/mailman/listinfo/criu);
-* For historical reasons we do not accept PRs, instead [patches are welcome](http://criu.org/How_to_submit_patches);
+* We accept github pull-requests and this is the preferred way to contribute to CRIU. If you prefer to send patches by email, you are welcome to send them to [the devel list](http://criu.org/How_to_submit_patches);
```
Comment (kolyshkin) [inline README.md]: s/pull-requests/pull requests/

**Pull request merged**